packages/react-components/react-avatar/src/components/AvatarGroupItem/AvatarGroupItem.test.tsx

```diff
@@ -1,25 +1,53 @@
 import * as React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import { AvatarGroupItem } from './AvatarGroupItem';
+import { AvatarGroupContext } from '../../contexts';
 import { isConformant } from '../../common/isConformant';
 
+const testId = 'testId';
+
+const ContextWrapper: React.FC = ({ children }) => (
+  <AvatarGroupContext.Provider value={{ isOverflow: true }}>{children}</AvatarGroupContext.Provider>
+);
+
 describe('AvatarGroupItem', () => {
   isConformant({
     Component: AvatarGroupItem,
     displayName: 'AvatarGroupItem',
-    // TODO: enable component-has-static-classnames-object
-    disabledTests: [
-      'component-has-static-classname',
-      'component-has-static-classname-exported',
-      'component-has-static-classnames-object',
-    ],
+    disabledTests: ['component-has-static-classname', 'component-has-static-classname-exported'],
     primarySlot: 'avatar',
+    renderOptions: {
+      wrapper: ContextWrapper,
+    },
+  });
+
+  it('renders only an avatar when context provides true for isOverflow', () => {
```

Comment:
Does the description need to change or the test? The context is providing `false` for `isOverflow`.